src/scripts/main-nav.js

```diff
@@ -130,7 +130,7 @@ function reflectUserState (id, nickname, imageurl, customLogoutCall) {
     strVar += ' <div class="login-txt"> ' + label + ' <\/div>';
     strVar += '</a>';
     strVar += '<ul class="dropdown-menu">';
-    strVar += '  <li><a href="/developers/">Dashboard</a></li>';
+    strVar += '  <li><a href="/dashboard">Dashboard</a></li>';
```

Comment:
Is this the only `/developers/` reference on that page?

Also, should we be directing to `/dashboard/` or `/dashboard`?   Need to be consistent for SEO.